mods/ctf/ctf_stats/chat.lua

```diff
@@ -183,6 +183,9 @@ minetest.register_chatcommand("transfer_rankings", {
 		if not ctf_stats.players[dest] then
 			return false, "Player '" .. dest .. "' does not exist."
 		end
+		if ctf.player(src) == ctf.player(dest) then
+			return false, "You cannot transfer your own rankings!"
```

Comment:
I'd prefer to avoid active voice here, because the caller needn't be the same as the player whose stats are being transferred. Instead, I'd prefer something similar to the following:

```suggestion
			return false, "Source name and destination name cannot be the same!"
```